templates/opensshd.conf.j2

```diff
@@ -224,3 +238,24 @@ PasswordAuthentication no
 PermitRootLogin no
 X11Forwarding no
 {% endif %}
+
+# Group matching configuration
+# ============================
+
+{% if ssh_server_match_group %}
```

Comment:
The heading `# Group matching configuration` could be included in the condition to not print empty headings. Something like:

```YAML
{% if ssh_server_match_group %}

# Group matching configuration
# ============================
```